test/unit/hooks/countryBan/countryBan.test.js

```diff
@@ -54,13 +52,61 @@ describe('countryBan', () => {
       server: sails.testServer,
       type: 'playerConnected'
     };
-    const handleCountryBan = sails.hooks.countryban.handleCountryBan;
-    const notifSpy = sandbox.spy(sails.hooks.discordnotifications, 'sendNotification');
+
+    handleCountryBan = sails.hooks.countryban.handleCountryBan;
+    notifSpy = sandbox.spy(sails.hooks.discordnotifications, 'sendNotification');
+    sandbox.stub(sails.helpers.sdtdApi, 'executeConsoleCommand').callsFake();
+  });
+
+  it('Kicks a player that connects from a listed country in blacklist mode', async () => {
+
+    await handleCountryBan(connectedMessage);
+
+    expect(notifSpy).to.have.been.calledOnce;
+
```

Comment:
Should probably add an assertion to check if `sails.helpers.sdtdApi.executeConsoleCommand` was called with the correct arguments as well. Same for the other tests, right now you're mostly testing the notification (which is okay) but not the actual action of kicking the player

Reply:
Will do, I was just following the test that was there previously is all